.yarnrc

```diff
@@ -0,0 +1 @@
+save-prefix false
```

Comment:
when i added the `js-cookie` dependency, I saw that it set a caret-version, so I went ahead and added this config file so that yarn will set explicit dependency versions in the future.